### ISSUE TITLE
Mute hashtags

### ISSUE THREAD

**Issue by Landsil** [pixelfed/pixelfed#5293]
Hi,
Currently Mastodon allows for muting of hastags, same feature would be amazing in Pixelfed.

eg. I used the service mostly for photography and would love to move various "ai" hashtags.

I've been manually muting users but limit is 250 accounts, now I block them, not sure if similar limit is in place there too but muting whole hashtags would be a much better user experience.

**Comment by ergonomic-paragraph** [comment]
Implementing the Mastodon-style muting would be amazing.

I’d love to be able to filter out all of the AI, furry, and porn content. Not having the ability to hide this stuff _really_ pushes me away from using Pixelfed.

**Comment by mattcoady** [comment]
Yes please! I'm trying to curate a good feed but it makes me not want to follow hashtags at all because you can't fine tune the content.

**Comment by heavyelement** [comment]
I'd like to chime in here and add my support to this. I'd love to be able to mute hashtags that are offensive like AI or other stuff.

**Comment by shleeable** [comment]
This is on the TODO list https://github.com/pixelfed/pixelfed/issues/5794

**Comment by liquidsunshine** [comment]
> This is on the TODO list [#5794](https://github.com/pixelfed/pixelfed/issues/5794)

Hi!  Have there been any updates for this?  It looks like the linked issue has been in QA for nearly a year.  If there's a way to contribute, I'd be happy to try -- I know myself and others would really appreciate having this feature.  :)

**Comment by MxL-spec** [comment]
This would be so important
